### PR TITLE
Uses optimized /projects?homepage=true endpoint and eliminates unnecessary API calls

### PR DESCRIPTION
**Pull request by parrish**
:warning: This depends on zooniverse/Panoptes#1552.  Don't merge until that's live.

### CODE REVIEW

**Comment by camallen** [comment]
api for this should be deployed now, be nice to see how much faster it is.

**Comment by parrish** [comment]
Do you want it to automatically use zooniverse/panoptes#1552?

If not, then we should remove the homepage query param and just let it come from the URL if we want to test it.

**Comment by camallen** [comment]
it'd be nice to just test the different end points manually and then decide which one is better. I was just letting folk know the staging api is ready for testing this.

**Comment by camallen** [comment]
this is really fast, i've tested on both homepage=true and cards=true. card needs the `http://` prefix added to avatar_src for reference.

**Comment by parrish** [comment]
> card needs the http:// prefix added to avatar_src for reference

Protocol relative urls ought to be fine, shouldn't they?  I had left it relative since it was broken in a HTTP local environment.  Not terribly picky here though.

**Comment by camallen** [comment]
you're right - i should just be prefixing the avatar url like you are in the fast serializer with a `//`

File: app/pages/projects.cjsx [hunk 1]
```diff
@@ -22,14 +22,12 @@ module.exports = React.createClass
     query = {include: 'avatar'}
 
     if !apiClient.params.admin
-      query.launch_approved = true
+      query.homepage = true
```
Comment (brian-c) [inline app/pages/projects.cjsx]: What's `homepage` mean here? This is the projects page. Still looks like the expected response though.

Comment (camallen) [inline app/pages/projects.cjsx]: an api param override to select the fast serializer. there is another that i've got called `?cards=true` as well.

Comment (parrish) [inline app/pages/projects.cjsx]: Yeah, it's a dumb query param name.

**Comment by parrish** [comment]
Now just waiting on zooniverse/Panoptes#1581 to merge/deploy

**Comment by camallen** [comment]
this should be good to go now

**Pull request merged**